### ISSUE TITLE
meshio is throwing TypeError: write() got an unexpected keyword argument 'binary'

### ISSUE THREAD

**Issue by mscroggs** [bempp/bempp-cl#2]
When plotting a GridFunction using the gmsh or paraview backend, I get this error.

Minimal code to cause error:

```python
import bempp.api
import numpy as np

bempp.api.PLOT_BACKEND = "gmsh"

grid = bempp.api.shapes.sphere(h=0.5)
space = bempp.api.function_space(grid, "DP", 0)

f = bempp.api.GridFunction(space, coefficients=np.ones(space.global_dof_count))

f.plot()
```

Commenting out this line stops the error: https://github.com/bempp/bempp-cl/blob/c649efd8627e89f0ce042527b1bcf39a6f7f0b28/bempp/api/grid/io.py#L142

**Comment by tbetcke** [comment]
This is due to recent parameter name changes in meshio and will be fixed in the next patch release.

**Comment by tbetcke** [comment]
Fixed in version 0.1.1
